src/Microsoft.Health.Fhir.Core/Features/Operations/Export/ExportJobTask.cs

```diff
@@ -489,6 +497,18 @@ private bool IsAnonymizedExportJob()
             return !string.IsNullOrEmpty(_exportJobRecord.AnonymizationConfigurationLocation);
         }
 
+        private string ExtractExportTaskLoggingMessage()
+        {
+            string id = _exportJobRecord?.Id ?? string.Empty;
```

Comment:
nit: Are the null checks for the `_exportJobRecord` necessary since it will never be null.

Reply:
Remove check on the record. Thanks!